### PR TITLE
Support RTL for setup screen

### LINKED ISSUE #3515: RTL support for setup page
I will make fix this once PR #3457 is merged, this ~~may~~ related to ~~#3015 and~~ https://github.com/OrchardCMS/OrchardCore/pull/3484#discussion_r278715401

### PR DESCRIPTION
**Pull request by hishamco**
Fixes #3515

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Setup/OrchardCore.Setup.csproj [hunk 1]
```diff
@@ -5,6 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\OrchardCore\OrchardCore.DisplayManagement\OrchardCore.DisplayManagement.csproj" />
```
Comment (sebastienros) [inline src/OrchardCore.Modules/OrchardCore.Setup/OrchardCore.Setup.csproj]: I don't think it's necessary

Comment (hishamco) [reply src/OrchardCore.Modules/OrchardCore.Setup/OrchardCore.Setup.csproj]: What about orchard helper?!!

Comment (sebastienros) [reply src/OrchardCore.Modules/OrchardCore.Setup/OrchardCore.Setup.csproj]: Ok

**Comment by sebastienros** [comment]
Didn't you forget the css/js assets? js assets might have changed due to Bootstrap 4.3.1 change, so you need to merge dev in your branch before building.

**Comment by hishamco** [comment]
OK .. thanks

**Comment by hishamco** [comment]
🕐📆

**Pull request merged**